### PR TITLE
Katello 3.7 docs

### PR DESCRIPTION
**Pull request by jturel**
First release, so careful review is appreciated.

- API docs not updated yet
- Upgrade steps not documented yet

Does this stay open until 3.7 is released? When is the appropriate time to address the above points?

### CODE REVIEW

**Comment by orrabin** [comment]
@jturel please rebase, there are conflicts.

File: _layouts/plugins/katello/documentation.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 pagename: Plugin Manuals
-versions: [{name: 3.6, displays: '3.6'}, {name: 3.5, display: '3.5'}, {name: 3.4, display: '3.4'}, {name: 3.3, display: '3.3'}, {name: 3.2, 'display': '3.2'}, {name: 3.1, display: '3.1'}, {name: 3.0, display: '3.0'}, {name: 2.4, display: '2.4'}, {name: 'nightly', display: 'nightly'}]
+versions: [{name: 3.7, display: '3.7'}, {name: 3.6, displays: '3.6'}, {name: 3.5, display: '3.5'}, {name: 3.4, display: '3.4'}, {name: 3.3, display: '3.3'}, {name: 3.2, 'display': '3.2'}, {name: 3.1, display: '3.1'}, {name: 3.0, display: '3.0'}, {name: 2.4, display: '2.4'}, {name: 'nightly', display: 'nightly'}]
```
Comment (ehelms) [inline _layouts/plugins/katello/documentation.html]: You'll want to update this to `display: '3.7-RC'` so folks don't think it's released yet.

**Comment by ehelms** [comment]
@jturel Please rebase

**Comment by jturel** [comment]
This is on my radar but haven't been in a hurry to get to it due to the release timeline so far...

**Comment by jturel** [comment]
@ehelms rebased

**Review by ehelms — APPROVED** [review]

**Comment by ehelms** [comment]
@jturel There is an error for what may be a missing file.

**Comment by jturel** [comment]
@ehelms fixed

this might be useful too https://github.com/theforeman/tool_belt/pull/84

**Pull request merged**